cgmanifest.json

```diff
@@ -486,7 +486,7 @@
         "other": {
           "name": "aspell",
           "version": "0.60.8",
-          "downloadUrl": "ftp://ftp.gnu.org/gnu/aspell/aspell-0.60.8.tar.gz"
+          "downloadUrl": "https://github.com/GNUAspell/aspell/archive/rel-0.60.8.tar.gz"
```

Comment:
When I wget and sha256sum the tar at this url, I get "242f8ffac41db7b7dbb08e1b3e6ea3abb9e8b71ffebe2931c009baee692ff8e3", which is different from the hash in your .signatures.json

Reply:
My bad. I'll update

```sh
$ curl -sL https://github.com/GNUAspell/aspell/archive/rel-0.60.8.tar.gz | sha256sum
242f8ffac41db7b7dbb08e1b3e6ea3abb9e8b71ffebe2931c009baee692ff8e3  -
```